README.md

```diff
@@ -99,36 +99,56 @@ The content can be accessed through each specific README link below.
     - Git
 - [Class 02](classes/02class/README.md)
     - Agile Methodologies
-    - AWS Network Deep Dive
+    - Networking Introduction
+    - AWS VPC
 - [Class 03](classes/03class/README.md)
     - AWS EC2
         - Load Balancing
         - Auto Scaling Group
+    - AWS CloudWatch
     - AWS Route 53
     - Container - Docker
         - Repositories
         - Images
         - Dockerfile
         - AWS ECS
+        - AWS ECR
 - [Class 04](classes/04class/README.md)
-    - Project #1 Kick-off
-    - AWS - IAM
-    - AWS - CLI
-    - AWS - S3
-    - AWS - RDS
-    - IaC - Terraform
+    - AWS IAM (Identity and Access Management)
+    - AWS CLI
+    - AWS S3 (Simple Storage Service)
+      - Website
+      > - AWS Cloudfront
+      > - AWS WAF (Web Application Firewall)
     - MakeFile
+    - IaC - Terraform
+    - AWS RDS (Relational Database Service)
+    - Project #1 Kick-off
 - [Class 05](classes/05class/README.md)
     - CI/CD Concepts
+      - AWS CodeBuild
     - Pipelines
-    - 3 Musketeers
+      - AWS CodePipeline
+      - AWS CodeDeploy (with Terraform)
+      - Github Actions ??
```

Comment:
Any comments on Github Actions? @kikobr82 @caiocezart

Reply:
I have played with it recently at work..
We can use it as a CI/CD tool.. it seems a good option ...

Reply:
Let's move on with GitHub Actions. we can have the following for class05:
- 3 Musketeers
 - Docker
 - Make
 - Compose
- CI/CD Concepts
  - GitHub Actions